docs/Git_Reference.md

```diff
@@ -0,0 +1,117 @@
+|--------|-------|--------------|
+| config | List the current config file |  git config --list |
+| | Configure the author name and email address to be used with your commits. Note that Git strips some characters (for example trailing periods) from user.name.| git config --global user.name "Sam Smith"|
+| | |git config --global user.email sam@example.com|
+| | |git config --global http.proxy http://my-proxy.com |
+| | | git config --global https.proxy https://my-proxy.com |
+| | Alias for pull to do a fetch + rebase (no merge) | pull.rebase=true |
+|Create a new local repository| | git init|
+| Check out a repository | Create a working copy of a local repository:	| git clone /path/to/repository |
+| | For a remote server, use:	| git clone username@host:/path/to/repository|
+| Add files |  Add one or more files to staging (index):|git add filename |
+| | Track new files |git add * |
+| |  | git add *.c |
+| | | git add file.c |
+| Commit | Commit changes to head (but not yet to the remote repository):	| git commit -m "Commit message"|
+| | Commit any files you've added with git add, and also commit any files you've changed since then:	| git commit -a |
+| | Commit and signoff, opens editor to add comments | git commit -s |
+| | Commit without running precommit tests | git commit -n |
+| | To modify the comment of your last commit, before a push | git commit —amend |
+| Squash commits | Squash all commits into one before a push | git log --oneline |
+| | Interactively | git rebase -i (commit hash) |
+| | Squash all commits | git rebase - root -i |
+| Push | Send changes to the master branch of your remote repository:	| git push origin master|
+| Status |	List the files you've changed and those you still need to add or commit:	|git status|
+| | Short status | git status -s |
+| diff | What changes were made | git diff  file.c |
+| | | git diff  --staged |
+| | For a list of files to be pushed | git diff --stat --cached [remote/branch] |
+| rm | Remove a file | git rm file.c |
+| mv | rename a file | git mv file1.c  file2.c |
+|Connect to a remote repository| If you haven't connected your local repository to a remote server, add the server to be able to push to it:|git remote add origin <server>|
+| Log | List all the commits to this repo | git log |
+| | List files commited to a branch | git log --oneline |
+| | List the files that were changed in each commit | git log --stat |
+| | | git log —names-only |
+| | |git —name-status |
+| | List changeset from 2 commits ago | git log  -p  -2 |
+| | Format the logs | git log  --pretty=format:"%h  %s"  --graph | 
+| | List commits since 2 weeks ago | git log  --since=2.weeks|
+| | List all currently configured remote repositories:|	git remote -v|
+|Branches| Create a new branch and switch to it:	| git checkout -b <branchname>|
+| | Switch from one branch to another:	|git checkout <branchname>|
+| | List all the branches in your repo, and also tell you what branch you're currently in:	|git branch --all |
```

Comment:
Your dashes are getting mangled in the docs pages such that when you try to copy them it doesn't work as expected.

Make sure all code examples are surrounded by backticks. I.E.

git branch --all  vs `git branch --all`

Reply:
Adde single backticks around all git commands.